sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/changestreams/dao/BigtableChangeStreamAccessor.java

```diff
@@ -52,6 +52,7 @@ public class BigtableChangeStreamAccessor implements AutoCloseable {
   // Create one bigtable data/admin client per bigtable config (project/instance/table/app profile)
   private static final ConcurrentHashMap<BigtableConfig, BigtableChangeStreamAccessor>
       bigtableAccessors = new ConcurrentHashMap<>();
+  private static Duration readChangeStreamTimeout = Duration.ofSeconds(15);
```

Comment:
should this be null?  DEFAULT_READ_CHANGE_STREAM_TIMEOUT  already defines the default.

Reply:
@tonytanger

Reply:
Sorry for the late reply.

I updated the location of the default value into the Accessor. This makes it more clear what the default value is and where it is set. BigtableIO, instead of setting it, will override the default timestamp if necessary.